migrations/tables/app_downstream_version.yaml

```diff
@@ -41,5 +41,10 @@ spec:
         type: text
       - name: preflight_result_created_at
         type: timestamp without time zone
+      - name: preflight_ignore_permissions
+        type: boolean
+        default: "false"
+        constraints:
+          notNull: true
```

Comment:
fyi, this won't migrate....  we don't yet support migrations to default w/ not null in schemahero.

i don't know if we need this to migrate?